src/Resources/skeleton/controller/Controller.tpl.php

```diff
@@ -13,9 +13,7 @@ class <?= $class_name; ?> extends Controller
     public function index()
     {
 <?php if ($twig_installed) { ?>
-        return $this->render('<?= $template_name ?>', [
-            'controller_name' => '<?= $class_name ?>',
```

Comment:
There is no need to "pass" a variable since we can "fill" every variables in the "final" template.
It avoids some "WTF" effect at "runtime": I did not understand why some variables were passed to the template from the controller, and some were hardcoded in the "final" template
Now everything is hardcoded in the "final" template. It's simpler IMHO because everything (magic ?) is in the same place.

Now: The "final" controller could be kept as it, and the "final" template should be reworked by the end user. It seemes quite legit to me.

*Note:* Sorry about all the quote, I don"t know how to easily distinguish the "final"/compiled template from the .tpl.

Reply:
maybe still useful as an example?

Reply:
Before, the link was: `subl://src/Controller/FooController.php:0`
now the link is: `subl:///home/gregoire/dev/trainings/2018-06-12/application/src/Controller/FooController.php:15`
Obviously, this example is specific to my computer !

Reply:
I think @nicolas-grekas referees to the passing of the controller name. I also agree that it would be nice to keep this part, because it shows a little bit how templates in Symfony work.

Reply:
Ahhh indeed I did not get it. Thanks for the clarification.